client/components/social-buttons/utils.ts

```diff
@@ -39,13 +39,18 @@ export const getRedirectUri = (
 		return `https://${ host + login( { socialService } ) }`;
 	}
 
+	let flow = 'start';
+	// TODO: I am restricting this to certain flows for testing sake, but I think this should be the default behavior.
+	if ( flowName === 'ai-site-builder' && socialService === 'github' ) {
+		flow = `setup/${ flowName }`;
+	}
+
+	let protocol = 'https';
```

Comment:
I think we can simply use `window.location.origin` instead of `host` and then we don't need to handle the protocal

Reply:
I am happy to think about this in another pr, but this is already improving the situation :)

Reply:
Sure 🙂